README.md

```diff
@@ -107,6 +107,18 @@ password: <token-value>
 ```
 Make sure to include the `pypi-` prefix for your token value.
 
+# Performance
+
+The following benchmarks were measured on the Rasberry Pi 4b (TODO: ensure the version is correct)
+
+The **Performance** column represents how long it takes to call one function, while the **Max Frequency** column represents how many times that function could be called every second.
+
+| Feature | Performance | Max Frequency | Function | Example | Desc |
```

Comment:
I think it would be better to rename `Max Frequency` to `Max Read Frequency`

Reply:
sure thing!

https://github.com/EdgePi-Cloud/edgepi-python-sdk/pull/423/commits/c4c1e3247936b686bbfc07d868db8c0eb74a3cc2